pkg/istio/rbac/v2/provider.go

```diff
@@ -78,6 +80,23 @@ func (p *v2) ConvertAthenzModelIntoIstioRbac(athenzModel athenz.Model, serviceNa
 			if err != nil {
 				continue
 			}
+
+			// Drop the query parameters from the HTTP path in the assertions due to the difference
+			// in the RBAC Envoy permissions config created by Authentication Policy and ServiceRole/ServiceRoleBindings.
+			// Which in case of,
+			// Authentication Policy - is created with a url_path object
+			// ServiceRole/ServiceRoleBindings - is created with a header object
+			re, err := regexp.Compile(`.*\?.*`)
+			if err != nil {
+				log.Errorln("Error while parsing the path regex: ", err.Error())
+				continue
+			}
+
+			if re.MatchString(path) {
```

Comment:
Can you add a simple test to cover path with query param?

Reply:
Sure, will do

Reply:
+1